pulp_python/app/serializers.py

```diff
@@ -1,30 +1,164 @@
+from gettext import gettext as _
+
 from rest_framework import serializers
 from pulpcore.plugin import serializers as platform
 
 from . import models
 
 
-class PythonContentSerializer(platform.ContentSerializer):
+class ClassifierSerializer(serializers.ModelSerializer):
+    """
+    A serializer for Python Classifiers
     """
-    A Serializer for PythonContent.
 
-    Add serializers for the new fields defined in PythonContent and
-    add those fields to the Meta class keeping fields from the parent class as well.
+    name = serializers.CharField(
+        help_text=_("A string giving a single classification value for a Python package.")
+    )
 
-    For example::
+    class Meta:
+        model = models.Classifier
+        fields = ('name',)
 
-    field1 = serializers.TextField()
-    field2 = serializers.IntegerField()
-    field3 = serializers.CharField()
 
-    class Meta:
-        fields = platform.ContentSerializer.Meta.fields + ('field1', 'field2', 'field3')
-        model = models.PythonContent
+class PythonPackageContentSerializer(platform.ContentSerializer):
+    """
+    A Serializer for PythonPackageContent.
     """
 
+    filename = serializers.CharField(
+        help_text=_('The name of the distribution package, usually of the format:'
+                    ' {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}'
+                    '-{platform tag}.{packagetype}')
+    )
+    packagetype = serializers.CharField(
+        help_text=_('The type of the distribution package '
+                    '(e.g. sdist, bdist_wheel, bdist_egg, etc)')
+    )
+    name = serializers.CharField(
+        help_text=_('The name of the python project.')
+    )
+    version = serializers.CharField(
+        help_text=_('The packages version number.')
+    )
+    metadata_version = serializers.CharField(
+        help_text=_('Version of the file format')
+    )
+    summary = serializers.CharField(
+        required=False, allow_blank=True,
+        help_text=_('A one-line summary of what the package does.')
+    )
+    description = serializers.CharField(
+        required=False, allow_blank=True,
+        help_text=_('A longer description of the package that can run to several paragraphs.')
+    )
+    keywords = serializers.ListField(
+        child=serializers.CharField(),
+        required=False,
+        help_text=_('A list of additional keywords to be used to assist searching for the '
+                    'package in a larger catalog.')
+    )
+    home_page = serializers.URLField(
+        required=False, allow_blank=True,
+        help_text=_('The URL for the package\'s home page.')
+    )
+    download_url = serializers.URLField(
+        required=False, allow_blank=True,
+        help_text=_('The URL from which this version of the package can be downloaded. ')
```

Comment:
+ from Pulp. In the past, there has been confusion about the `path` (where you can download this from the remote sourcel) and `download_url` (where you can download this from Pulp)